recipe/recipe.yaml

```diff
@@ -0,0 +1,56 @@
+context:
```

Comment:
What is this file doing here?

Reply:
nothing, has to be removed :) wanted to get a review before trigggering another build.